chart/values.yaml

```
@@ -17,6 +17,8 @@ openshift-gitops:
         requests:
           cpu: "1"
           memory: 3Gi
+  # TLS check can cause unexpected hanging during login: https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_login/#options
+  skip-tls: true
```

Despite the current state of ArgoCD for us, I'd still make the default `false`:
```suggestion
  skip-tls: false
```